.snyk

```diff
@@ -0,0 +1,22 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+version: v1.25.0
+ignore: {}
+patch: {}
+exclude:
+  global:
+    - '.openshift-ci'
+    - '.github/workflows/scripts/scanner-update-nvd.py'
+    - '.github/workflows/scripts/check-image-vulnerabilities.py'
```

Comment:
I assume specific files are added here (as opposed to the whole `.github` folder) is because snyk found specific issues in these files.  Since other top level folders are ignored, is there a significant difference here between ignoring specific files identified in the Snyk scan and entire folders?

Reply:
The ignore list is based on the 180 warnings that Snyk reported initially. My rule of thumb was to add specific files if <= 2 per directory, and add the directory if issues were detected in more than 2 files.